pkg/mimir/modules.go

```diff
@@ -1024,6 +1031,54 @@ func (t *Mimir) initUsageStats() (services.Service, error) {
 	return t.UsageStatsReporter, nil
 }
 
+func (t *Mimir) initContinuousTest() (services.Service, error) {
+	start := func(ctx context.Context) error {
+		logger := util_log.Logger
+		var err error
+
+		level.Info(logger).Log("msg", "initing continuous-test")
```

Comment:
I think _most_ of this start method should be removed since the type of stuff it's configuring already exists in Mimir. E.g. the tracing, the prometheus registry, the instrumentation server.